.travis.yml

```diff
@@ -48,10 +48,44 @@ script:
 - operator-sdk generate k8s
 - operator-sdk build quay.io/example/memcached-operator:v0.0.1
 - sed -e 's/imagePullPolicy. Always/imagePullPolicy. Never/g' deploy/operator.yaml
+# Run the operator as a pod
 - kubectl create -f deploy/rbac.yaml
 - kubectl create -f deploy/crd.yaml
 - kubectl create -f deploy/operator.yaml
 - kubectl get deployment
+# Create memcached cr
+- |
+  echo "apiVersion: \"cache.example.com/v1alpha1\"" > deploy/cr.yaml
+- |
+  echo "kind: \"Memcached\"" >> deploy/cr.yaml
+- echo "metadata:" >> deploy/cr.yaml
+- |
+  echo "  name: \"example-memcached\"" >> deploy/cr.yaml
+- echo "spec:" >> deploy/cr.yaml
+- |
+  echo "  size: 3" >> deploy/cr.yaml
+- kubectl apply -f deploy/cr.yaml
+- sleep 30s
+- kubectl get deployment
+- kubectl get pods
+- kubectl get memcached/example-memcached -o yaml
+# Expand to size to 4
+- |
+  echo "apiVersion: \"cache.example.com/v1alpha1\"" > deploy/cr.yaml
+- |
+  echo "kind: \"Memcached\"" >> deploy/cr.yaml
+- echo "metadata:" >> deploy/cr.yaml
+- |
+  echo "  name: \"example-memcached\"" >> deploy/cr.yaml
+- echo "spec:" >> deploy/cr.yaml
+- |
+  echo "  size: 4" >> deploy/cr.yaml
+- kubectl apply -f deploy/cr.yaml
+- sleep 30s
```

Comment:
this might be a little too much sleep time right? Just worried about the overall time the test will take.

Reply:
Yeah don't use sleep at all. Better pull an api until it is ready.